lib/urbanopt/geojson/helper.rb

```diff
@@ -207,9 +209,35 @@ def self.floor_print_from_polygon(polygon, elevation, origin_lat_lon, runner, zo
           floor_print = OpenStudio.reverse(floor_print)
           runner.registerWarning('Reversing floor print')
         end
+
+        #check for scaling
+        if scaled_footprint_area > 0
+
+          # check that the scaled_footprint_area desired is no less than X % of the original
+          original_floor_print_area = OpenStudio::getArea(floor_print).get
+
+          if scaled_footprint_area / original_floor_print_area <= 0.5 || scaled_footprint_area / original_floor_print_area >= 2
+            # TOO MUCH SCALING...using original footprint when scaled is 2x bigger or smaller than the original
+            runner.registerWarning("Desired scaled_footprint_area is a factor of 2 of the original footprint...keeping original footprint (no scaling!)")
```

Comment:
@kflemin , the code is working correctly for this condition, but is not scaling to the new area in the else statement.